Doc/whatsnew/3.7.rst

```diff
@@ -288,7 +288,7 @@ importlib.resources
 This module provides several new APIs and one new ABC for access to, opening,
 and reading *resources* inside packages.  Resources are roughly akin to files
 inside of packages, but they needn't be actual files on the physical file
-system.  Module loaders can implement the
+system.  Module loaders can support the
```

Comment:
"support" used twice reads a little weird.  Maybe the first one can be "provide"?